frontend/components/Players/DraggableQueue.vue

```diff
@@ -41,7 +41,9 @@
             <like-button :item="item" />
           </v-list-item-action>
           <v-list-item-action v-if="!isPlaying(index)" class="mr-2">
-            <item-menu :item="item" />
+            <v-btn icon @click="removeFromQueue(item)">
```

Comment:
```suggestion
            <v-btn icon @click="playbackManager.removeFromQueue(item)">
```

Reply:
With what I suggest above:

```suggestion
            <v-btn icon @click="playbackManager.removeFromQueue(item.Id)">
```